orttraining/orttraining/python/training/__init__.py

```diff
@@ -2,6 +2,23 @@
 # Copyright (c) Microsoft Corporation. All rights reserved.
 # Licensed under the MIT License.
 # --------------------------------------------------------------------------
+
+
+# Verify proper PyTorch is installed before proceding to ONNX Runtime initializetion
+from packaging import version
+MINIMUM_TORCH_VERSION_STR = '1.8.1'
+try:
+    import torch
```

Comment:
Is this the right place to raise the exception? IIUC, this would prevent someone from using `ORTTrainer` with an older version of PyTorch, which is/should be allowed (at least until we formally drop support for the old API).

Perhaps the right place to put it is in the `torch_ort` wrapper library?

Reply:
Internal users are still not using torch_ort and using older pytorch, wasting days of developing on the incorrect environment

Maybe lazy loading on ORTModule? move `ORTModule` on onnxruntime repo to `onnxruntime.training.ortmodule` namespace but keep it in the root for torch_ort?